server/server.go

```diff
@@ -329,17 +332,12 @@ func (s *IDPServer) realishEmail(email string) string {
 }
 
 // mustGenRSAKey generates an RSA key of the specified size
-// Migrated from legacy/tsidp.go:2307-2329
-func mustGenRSAKey(bits int) (kid uint64, k *rsa.PrivateKey) {
-	var err error
+func mustGenRSAKey(bits int) (kid uint64, k *rsa.PrivateKey, err error) {
```

Comment:
The name on this is misleading now `must` typically means "will panic if fails"


```suggestion
// genRSAKey generates an RSA key of the specified size
func genRSAKey(bits int) (kid uint64, k *rsa.PrivateKey, err error) {
```